### DISCUSSION TITLE
Is Rails supported?

### DISCUSSION THREAD

**Discussion by lucasfcunha** [coollabsio/coolify#1994] [Q&A] [answered]
I'm looking everywhere and I can't find a definitive answer if Rails is supported or not. I see that the author mentioned that it would be supported on v4 but I can't find anything that says that this is already the case.

Can someone clarify to me if Rails is supported? Thanks!

**Comment by maietta** [comment, marked as answer]
Yes.

You can use a Dockerfile and deploy using that.

Anything that can run in a docker container can be run with a Dockerfile crafted for your app.
